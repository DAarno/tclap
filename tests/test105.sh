#!/bin/sh

../examples/test37 -h -a 5 > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test105.out; then
	exit 0
else
	exit 1
fi
