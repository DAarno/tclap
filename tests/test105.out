#compdef test37

# test37 version none

_arguments -s -S \
  '(-c --c_int)'{-c,--c_int}'[int arg]:int' \
  '(-a --a_int)'{-a,--a_int}'[int arg]:int' \
  '(-)--version[displays version information and exits]' \
  '(-)'{-h,--help}'[displays usage information and exits]'
